src/stepfunctions/steps/service.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, state_id, **kwargs):
             heartbeat_seconds_path (str, optional): Path specifying the state's heartbeat value in seconds from the state input. When resolved, the path must select a field whose value is a positive integer.
             input_path (str, optional): Path applied to the state’s raw input to select some or all of it; that selection is used by the state. (default: '$')
             parameters (dict, optional): The value of this field becomes the effective input for the state.
+            result_selector (dict, optional): The value of this field becomes the effective result of the state.
```

Comment:
Do we want to add result_selector for `EventBridgePutEventsStep` as well?

Reply:
fixed in next commit